jedi/api/environment.py

```diff
@@ -206,6 +220,33 @@ def _get_executable_path(path, safe=True):
     return python
 
 
+def _get_executables_from_windows_registry(version):
+    # The winreg module is named _winreg on Python 2.
+    try:
+      import winreg
+    except ImportError:
+      import _winreg as winreg
+
+    # TODO: support Python Anaconda.
+    sub_keys = [
+      r'SOFTWARE\Python\PythonCore\{version}\InstallPath',
+      r'SOFTWARE\Wow6432Node\Python\PythonCore\{version}\InstallPath',
```

Comment:
WTF is `Wow6432Node` lol. I also see it in CPython's source code. You don't have to explain, I'm just a bit surprised that this is in there.